storybook/index.html

```diff
@@ -72,62 +72,13 @@ <h1 class="sectionHeading">Hello</h1>
     </section>
 
     <section class="demo">
-      <div class="scrollCarousel">
-        <div class="scrollCarousel--contents">
-         <div class="mover-1">
```

Comment:
the removals are unexpected -- you might need to update master? not sure but we can figure it out. I don't think the PR should have any removals though

Reply:
(I merged master and put these bits back in)